### PR TITLE
Cmiyoung documentation jsspric

### PR DESCRIPTION
**Pull request by jsspric**
*Issue #, if available:*

*Description of changes:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-jumpstart-industry-pack/blob/main/CONTRIBUTING.md) doc
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-jumpstart-industry-pack/blob/main/CONTRIBUTING.md#committing-your-change)
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have updated any necessary [documentation](https://github.com/aws/sagemaker-jumpstart-industry-pack/blob/main/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/source/what-is.rst [hunk 1]
```diff
@@ -23,20 +23,20 @@ in the ML lifecycle.
...
    the processing containers to the Amazon S3 bucket.
    SageMaker terminates the processing job and its resources.
 #. You can download the result from the Amazon S3 bucket to the Studio notebook kernel
    and you can start training pretrained language models,
    such as BERT and its variants.
-#. You evaluate the model performance and start using the model for making prediction.
+#. You evaluate the model performance and start using the model for making predictions.
 
 
 What It Does
```
Comment (mchoi8739) [inline docs/source/what-is.rst:42]: Should this be lower-cased as well?

File: docs/source/what-is.rst [hunk 2]
```diff
@@ -46,13 +46,13 @@ The library provides API operations to process financial multimodal
...
+- It calculates NLP scores for the SEC filings text data.
 - It summarizes text data using the :class:`~smjsindustry.Summarizer` class,
   choosing between the Jaccard and k-medoids algorithms.
 - It combines text data, tabular data, and categorical data into a multimodal dataset.
 - It provides pretrained RoBERTa-SEC language models with
-  S&P 500 10-K/Q filings over the last decades and the English wikipedia corpus.
+  S&P 500 10-K/Q filings over the last decades and the English Wikipedia corpus.
 
 
 What It Offers
```
Comment (mchoi8739) [inline docs/source/what-is.rst:58]: Should this be lower-cased as well?

File: src/smjsindustry/finance/nlp_score_type.py [hunk 3]
```diff
@@ -27,7 +27,7 @@ class NLPScoreType:
     to perform NLP scoring using SageMaker JumpStart Industry's internal word lists.
 
     A single ``NLPScoreType`` or a list of ``NLPScoreTypes`` is required
-    when initializing an :class:`~smjsindustry.NLPScorerConfig`. Passing the data required by
+    when initializing a :class:`~smjsindustry.NLPScorerConfig`. Passing the data required by
```
Comment (mchoi8739) [inline src/smjsindustry/finance/nlp_score_type.py:30]: The doc renders like the following image:
Screen Shot 2021-09-29 at 3 03 30 PM

I think it should still be "an"?

File: docs/source/text-summarizer-algos.rst [hunk 4]
```diff
@@ -13,9 +13,9 @@ and :math:`m \le n`.
 
 In general, there are two categories of text summarization:
 **extractive summarization** and **abstractive summarization**. If for
-each :math:`i` in :math:`[0, m), S’_i` is in :math:`{S_0, S_1, ...,
+each :math:`i` in :math:`[0, m), and S’_i` is in :math:`{S_0, S_1, ...,
```
Comment (mchoi8739) [inline docs/source/text-summarizer-algos.rst:16]: in :math:`[0, m), and S’_i`
-->
in :math:`[0, m)`, and :math:`S’_i`

File: docs/source/text-summarizer-algos.rst [hunk 5]
```diff
@@ -13,9 +13,9 @@ and :math:`m \le n`.
...
 S_{n-1}}`, the summarization is **extractive summarization**. If there
-exists :math:`j` in :math:`[0, m), S’_j` is not in :math:`{S_0, S_1, ..., S_{n-1}}`, the
+exists :math:`j` in :math:`[0, m), and S’_j` is not in :math:`{S_0, S_1, ..., S_{n-1}}`, the
```
Comment (mchoi8739) [inline docs/source/text-summarizer-algos.rst:18]: in :math:`[0, m), and S’_i`
-->
in :math:`[0, m)`, and :math:`S’_i`

**Comment by mchoi8739** [comment]
nit comments. I can fix those as a fast follow-up

**Review by mchoi8739 — APPROVED** [review]

**Review by sophiayue1116 — APPROVED** [review]

File: src/smjsindustry/finance/processor_config.py [hunk 6]
```diff
@@ -224,7 +224,7 @@ class KMedoidsSummarizerConfig(FinanceProcessorConfig):
     k-medoids minimizes the sum of dissimilarities between vectors in a cluster
     and one of the vectors designated as the representative of that cluster;
     the representative vectors are called medoids. The collection of medoids
-    and the m sentences in the document closest to the cluster medoids is
+    and the m sentences in the document closest to the cluster medoids are
```
Comment (johnhe-dev) [inline src/smjsindustry/finance/processor_config.py:227]: and the m sentences in the document corresponding to the cluster medoids

Comment (mchoi8739) [reply src/smjsindustry/finance/processor_config.py:227]: Will fix it in a follow-up PR.

File: docs/source/what-is.rst (hunk 2 above)
Comment (jsspric) [reply docs/source/what-is.rst:58]: Yes.

File: src/smjsindustry/finance/nlp_score_type.py (hunk 3 above)
Comment (jsspric) [reply src/smjsindustry/finance/nlp_score_type.py:30]: Oh yes, sorry! Didn't know that was how it rendered.

File: docs/source/what-is.rst (hunk 1 above)
Comment (jsspric) [reply docs/source/what-is.rst:42]: Yes.

**Pull request merged**